### ISSUE TITLE
error in `flash` with `ebnm_pn'  4 out of 20 runs have this problem. 

### ISSUE THREAD

**Issue by NKweiwang** [stephenslab/ebnm#2]
I tried `flash` with `ebnm_pn` for GTEx zscore data for missing value imputation (10 fold OCV).
4 out of 20 runs have the following problems.
``` Error in optim(startpar, function(par, x, s) { : 
  L-BFGS-B needs finite values of 'fn'
In addition: There were 30 warnings (use warnings() to see them)
Error in mle_normal_logscale_grad(x, s) : 
  optim failed to converge; debug information saved to temp_debug.RDS  ```

please take a look at the debug.rds data.

Thanks a lot.

[temp_debug.RDS.zip](https://github.com/stephenslab/ebnm/files/1656865/temp_debug.RDS.zip)

**Comment by NKweiwang** [comment]
the temp_debug.RDS file doesn't seem readable. I just re-upload it again. 
This is the same data file.

[temp_debug.RDS.zip](https://github.com/stephenslab/ebnm/files/1656874/temp_debug.RDS.zip)

**Comment by stephens999** [comment]
problem was s=0 for some s.
Fixed in ba1c03d744be327523439cec9ea43b6391f7fa73
